### PR TITLE
updates for adding video library

### PR DESCRIPTION
**Pull request by darrell-thobe-sp**
Starting point for video library

### CODE REVIEW

File: src/pages/colab.js [hunk 1]
```diff
@@ -1,48 +1,174 @@
...
+            <div className={styles.contentContainer}>
+              <div className={styles.gettingStartedText}>
+                <div className={styles.gettingStartedOne}>
+                  CoLab Marketplace
+                </div>
+                <div className={styles.gettingStartedTwo}>
+                  What is the CoLab?
+                </div>
+                <div className={styles.gettingStartedThree}>
+                  Jump into our{' '}
```
Comment (jordan-violet-sp) [inline src/pages/colab.js:55]: @darrell-thobe-sp this text seems like it was left over and should be removed. If you read it on the built site in the PR, you'll see what I mean.

File: src/pages/colab.js [hunk 2]
```diff
@@ -1,48 +1,174 @@
...
+                  <span className={styles.bold}>
```
Comment (jordan-violet-sp) [inline src/pages/colab.js:56]: @darrell-thobe-sp can you unbold this body?

File: src/pages/blog.js [hunk 3]
```diff
@@ -8,33 +8,70 @@ import BlogBanner from '../components/blog/BlogBanner';
...
+      <main className={styles.main}>
+        <BlogBanner />
+        <div>
+          <div className={styles.mainCard}>
+            <div className={styles.contentContainer}>
+              <div className={styles.gettingStartedText}>
+                <div className={styles.gettingStartedOne}>Community Blog</div>
+
+                <div className={styles.gettingStartedThree}>
+                  <span className={styles.bold}>
```
Comment (jordan-violet-sp) [inline src/pages/blog.js:40]: @darrell-thobe-sp can you unbold this?

**Review by jordan-violet-sp — CHANGES_REQUESTED** [review]
@darrell-thobe-sp take a lot at some of the comments I made on some text.

**Review by tyler-mairose-sp — APPROVED** [review]

**Pull request merged**